configuration/messageproperties/metadata_en.properties

```diff
@@ -0,0 +1,4 @@
+# CES is using the "Confirmed" checkbox as if it meant "1a vez", which
+# more or less means "new diagnosis" or "first visit for this problem."
+# See https://pihemr.atlassian.net/browse/MEX-459
+coreapps.Diagnosis.Certainty.CONFIRMED=First time
```

Comment:
Are these repos on Transifex?

Reply:
Isn't this just the hacky First time/Confirmed fix you added? Not sure why it would need to be in Transifex?

Reply:
Sure, fair enough.